### PR TITLE
Added notes to Deepxde section and minor changes on github.rst

### PR DESCRIPTION
**Pull request by Ehaw04**
I will continue adding notes through the day but this PR is to finish my individual tasks

### CODE REVIEW

File: docs/source/deepxde.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 DeepXDE
 =======
 
+
+Phisics Informed Neural Netwoeks vs Finite Element Method
```
Comment (pescap) [inline docs/source/deepxde.rst:5]: check spelling

File: docs/source/deepxde.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@ DeepXDE
 =======
 
 
-Phisics Informed Neural Netwoeks vs Finite Element Method 
+Phisics Informed Neural Networks vs Finite Element Method
```
Comment (pescap) [inline docs/source/deepxde.rst:5]:
```suggestion
Physics-Informed Neural Networks (PINNs) vs Finite Element Method (FEM)
```

File: docs/source/deepxde.rst [hunk 3]
```diff
@@ -24,20 +26,20 @@ data.
 
 .. image:: ~Image/RedNeuronal.png
 
-As shown in the figure, the layer of neurons is represented with a nonlinear transformation. In each layer shown in Figure 1, the network uses the same activation function (non-linear transformation in Figure 2).
+As shown in the figure, the layer of neurons is represented with a nonlinear transformation. In each layer shown in figure 1, the network uses the same activation function (nonlinear transformation in figure 2).
```
Comment (pescap) [inline docs/source/deepxde.rst:29]:
```suggestion
As shown in the figure, the layer of neurons is represented with a non-linear transformation. In each layer shown in Figure 1, the network uses the same activation function (non-linear transformation in Figure 2).
```

**Comment by Ehaw04** [comment]
@pescap  I think this is ready to merge

**Comment by pescap** [comment]
Thank you!

**Pull request merged**